acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -1180,8 +1180,8 @@ void ocp_nlp_opts_set(void *config_, void *opts_, const char *field, void* value
             for (ii=0; ii<N; ii++)
                 config->dynamics[ii]->opts_set(config->dynamics[ii], opts->dynamics[ii], "compute_hess", value);
             // constraints TODO disabled for now as prevents convergence !!!
```

Comment:
This can be removed, then.
Maybe @giaf can comment, where these convergence issues occurred.
Just out of curiosity and because the constraint hessian contributions didnt actually change in this PR.

Reply:
@giaf were you referring to an example in particular?

Reply:
I don't remember now, it was long time ago

Reply:
I guess we can just enable it again, and keep an eye if issues arise somewhere

Reply:
great, thanks! I wait for Travis and merge.